tests/copp/test_copp.py

```diff
@@ -268,19 +314,13 @@ def disable_lldp_for_testing(
     """Disables LLDP during testing so that it doesn't interfere with the policer."""
     duthost = duthosts[rand_one_dut_hostname]
 
-    logging.info("Disabling LLDP for the COPP tests")
+    logging.info("Disabling LLDP Container auto-restart for the COPP tests")
```

Comment:
IIRC the reason we put this in a separate fixture was so that even if there was some issue during setup/teardown then this code should still execute and ensure that LLDP is brought back. Would it be possible to move the startup/shutdown of LLDP and BGP into this fixture and handle the multi-asic parts here as well?

Reply:
sure, ok  I will disable bgp and lldp feature itself  (which make sure all instance will be down in multi-asic)

Reply:
@daall Looks  like there is issue with LLDP where even if we stop the lldp the config reload will restart it . Looks like we don't need to stop LLDP then ?

Reply:
@daall I have confirmed fixture disable_lldp_for_testing where we disable lldp gets restarted as part of config reload/swap_syncd. So I am planning to remove this fixture as test cases works fine and passes without needing to disable lldp.

Reply:
@daall Updated based on my above comment.

Reply:
makes sense to me, thanks for checking!